h/traversal/organization.py

```diff
@@ -16,17 +11,16 @@ def __getitem__(self, pubid):
         if organization is None:
             raise KeyError()
 
-        # Inherit global ACL. See comments in :py:class`h.traversal.AuthClientRoot`.
-        organization.__parent__ = Root(self.request)
-
-        return organization
+        return OrganizationContext(organization, self.request)
 
 
 class OrganizationContext:
     """Context for organization-based views."""
 
     def __init__(self, organization, request):
-        # TODO Links service
+        # Inherit global ACL. See comments in :py:class`h.traversal.AuthClientRoot`.
+        self.__parent__ = Root(request)
```

Comment:
The root ACL's all seem to be administration related. Not sure what's up with that.

Reply:
FWIW I don't think we should bother with these `:py:class` things in docstrings or comments. They're a Sphinx thing and we aren't parsing these with Sphinx

Reply:
I agree, I was just copy pasting it about. I'll scoop them out when I see them from now.